apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/RevurderingRoutes.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.etterlatte.behandling.revurdering
+
+import io.ktor.http.HttpStatusCode
+import io.ktor.server.application.call
+import io.ktor.server.application.log
+import io.ktor.server.request.receive
+import io.ktor.server.response.respond
+import io.ktor.server.routing.Route
+import io.ktor.server.routing.application
+import io.ktor.server.routing.post
+import io.ktor.server.routing.route
+import no.nav.etterlatte.behandling.GenerellBehandlingService
+import no.nav.etterlatte.behandling.domain.toDetaljertBehandling
+import no.nav.etterlatte.libs.common.SAKID_CALL_PARAMETER
+import no.nav.etterlatte.libs.common.behandling.RevurderingAarsak
+import no.nav.etterlatte.libs.common.sakId
+
+internal fun Route.revurderingRoutes(
+    revurderingService: RevurderingService,
+    generellBehandlingService: GenerellBehandlingService
+) {
+    val logger = application.log
```

Comment:
Oi, denne hadde eg ikkje sett før. Kva tenkjer du om denne kontra LoggerFactory.getLogger?

Reply:
Virker som det er praksisen runt logging i routes. Hade varit muligt å bruke loggerfactory, men vi er ju i en funksjon så her kunde vi f.eks ikke brukt no klasse-navn for den loggern. Så da hade vi behøvt selv navngi alla de logsen?